lambdafcns/delete_tile_index_entry_lambda.py

```diff
@@ -0,0 +1 @@
+../lambda/delete_tile_index_entry_lambda.py
```

Comment:
Is this a move or a symlink?

Reply:
Symlink to allow tests. I'd like to rename the `lambda` folder, eventually.

Reply:
Looks like a symlink, be aware that they don't work for everyone. If the filesystem doesn't support symlinks then they cannot be created.